posthog/models/entity.py

```diff
@@ -89,6 +90,11 @@ class ExclusionEntity(Entity, FunnelFromToStepsMixin):
     def __init__(self, data: Dict[str, Any]) -> None:
         super().__init__(data)
 
+    @property
+    def sql_id(self):
+        # TODO: prevent SQL injection :(
```

Comment:
Probably just using a UUID here works.

Had other higher priority bugs to look into, will come back to this on Tuesday.